test/integration/multikueue/multikueue_test.go

```diff
@@ -456,6 +456,20 @@ var _ = ginkgo.Describe("Multikueue", func() {
 			}, util.Timeout, util.Interval).Should(gomega.Succeed())
 		})
 
+		ginkgo.By("changing the status of the jobset in the worker, updates the manager's jobset status", func() {
+			gomega.Eventually(func(g gomega.Gomega) {
+				createdJobSet := jobset.JobSet{}
+				g.Expect(worker2TestCluster.client.Get(worker2TestCluster.ctx, client.ObjectKeyFromObject(jobSet), &createdJobSet)).To(gomega.Succeed())
+				createdJobSet.Status.Restarts = 10
+				g.Expect(worker2TestCluster.client.Status().Update(worker2TestCluster.ctx, &createdJobSet)).To(gomega.Succeed())
+			}, util.Timeout, util.Interval).Should(gomega.Succeed())
+			gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
I don't see yet the flow, could you explain. If in the code we watch for updates on the workloads, how do we get notification about the remote job status update (`createdJobSet.Status.Restarts = 10`)?

Reply:
gotcha, we will prefer to use the origin label ultimately